src/components/stepforms/widgets/FilterSimpleCondition.vue

```diff
@@ -55,6 +55,8 @@ type LiteralOperator =
   | 'be less than or equal to'
   | 'be one of'
   | 'not be one of'
+  | 'matches pattern'
```

Comment:
I think that we should be clear here that we expect a regex pattern.
Also, regex are very powerful but very geeky and not in everybody's reach.
Should we not implement, aside from this generic regex matching, more basic 'contains', 'does not contain', 'begins with', 'does not begin with', 'ends with', 'does not end with' which are the most frequent use cases that we should target here